source/Libraries/HumbleLibrary/HumbleLibrary.cs

```diff
@@ -375,7 +384,7 @@ public override IEnumerable<PlayController> GetPlayActions(GetPlayActionsArgs ar
                 }
                 else
                 {
-                    var installed = GetInstalledGames().FirstOrDefault(a => a.GameId == args.Game.GameId);
+                    var installed = GetInstalledGames().FirstOrDefault(a => a.GameId == args.Game.GameId && a.IsInstalled);

```

Comment:
If a game is not detected as installed because its directory doesn't exists, then it should not be returned by GetInstalledGames() at all.

Reply:
Fixed in https://github.com/JosefNemec/PlayniteExtensions/pull/156/commits/b9fd091de8e5b9a911b9eb6b41ae58f05e785f03